config/locales/spotlight.en.yml

```diff
@@ -1,4 +1,9 @@
 en:
+  time:
+    formats:
+      date:
+        formats:
+          default: '%Y-%m-%d %l:%M%P'
```

Comment:
It's totally the right way to do it, it's just not appropriate for Spotlight to decide the default time format for someone else's application. Maybe `l time, format: :short` (out of the box: 18 Jan 06:10) is roughly equivalent? Or, again, we can always invent a new format key and use that.